packages/components/atoms/f-form-field/src/components/FormFieldAffixed.vue

```diff
@@ -53,7 +52,7 @@ export default {
     props: {
         attributes: {
             type: Object,
-            default: () => {}
+            default: () => ({})
```

Comment:
It means the default value returns an empty object rather than undefined:
![20220713T153949](https://user-images.githubusercontent.com/26894168/178761245-a2294299-cc3c-42b6-b5c1-a51a5f1675dd.png)